package.json

```diff
@@ -28,9 +28,9 @@
   },
   "homepage": "https://github.com/mervick/emojionearea#readme",
   "dependencies": {
-    "emojione": ">=2.0.1",
-    "jquery": ">=1.8.3",
-    "jquery-textcomplete": ">=1.3.4"
+    "emojione": "^3.1.2",
```

Comment:
Do not change the dependencies, because emojionearea can work with old versions too.